domain/credentials/service.go

```diff
@@ -46,28 +46,27 @@ func (s Service) ManageNutsOrgCredential(customer domain.Customer, shouldHaveCre
 		if customer.City == nil {
 			return fmt.Errorf("customer.City must be set for issuing a credential")
 		}
-		issue := true
-		for _, credential := range credentials {
-			if credential.Organization.Name == customer.Name && credential.Organization.City == *customer.City {
```

Comment:
So you now always recreate the credentials, even if name and city are unchanged?

Reply:
Yes